### ISSUE TITLE
Generate the label format of frame #,n[,id,x,y,w,h,label]

### ISSUE THREAD

**Issue by StrongerzZ** [darkpgmr/DarkLabel#8]
Hi,

I want to know how to modify the darklabel.yml file to generate the label format of frame #,n[,id,x,y,w,h,label] (previously existed in Darklable1.3_part1 version)
Thank you very much!

![image](https://user-images.githubusercontent.com/59546516/130027974-45221eb1-f3e7-4bcf-95da-63ebc9f94c12.png)

**Comment by darkpgmr** [comment]
Sorry but that label format is no longer supported after darklabel2.0.
The initial format is unnatural and hard to parse, so I decided to stop supporting it.
